include/circt/Dialect/FIRRTL/FIRRTLLowering.h

```diff
@@ -0,0 +1,23 @@
+//===- FIRRTLLowering.h - Definition for the shareable FIRRTL pipeline ----===//
```

Comment:
Nit: header description should probably not be same as `Firtool.h`.

Hyper-nit: these header blocks should have the filetype specifier (`- C++ -`) in headers.

Really wish these were generated/linted by a tool, how does LLVM not already have this, they're nice looking but a bit of a pain... :smile: .